src/components/playground/TypographyInput.js

```diff
@@ -145,32 +148,38 @@ function FontFamily(props) {
 
   return (
     <>
-      <TextField
+      <InputLabel id="label" style={{ fontSize: '0.8rem' }}>
+        Font Family
+      </InputLabel>
+      <Select
+        id="standard-select-font-native"
         variant="standard"
         label={label}
         value={value}
         error={error.length > 0}
         onChange={handleChange}
         helperText={error || 'Add Fontfamily from loaded families.'}
```

Comment:
Could you remove `helperText` since it is not a property on the `Select` component